api/src/main/java/org/pmiops/workbench/cohortbuilder/SearchGroupItemQueryBuilder.java

```diff
@@ -631,6 +647,9 @@ private static void addParamValueAndFormat(
               .collect(Collectors.toList());
 
       if (!parents.isEmpty() || Domain.DRUG.toString().equals(domain)) {
+        String domainParam =
```

Comment:
Question why is is in wrapped in if condition !parents.isEmpty()

Reply:
if we only have child nodes.. then the sql is simple
`is_standard = ? and concept_id in (?)`

if we have parents.. we have to lookup all parent and child nodes
```
is_standard = ? and concept_id in (
 select distinct c.concept_id
        from `project.dataset.cb_criteria` c
        join (select cast(cr.id AS string) as id
          from `project.dataset.cb_criteria` cr
          where (domain_id = ?
            and is_standard = ?
            and concept_id in (?)
            and is_selectable = 1 )) a
        on (c.path like concat('%.', a.id, '.%') or c.path like concat('%.', a.id) or c.path like concat(a.id, '.%'))
)
```

For Drugs we always have to lookup parent and child nodes.